From: Antonio Terceiro <terceiro@debian.org>
Date: Tue, 10 Oct 2017 15:08:43 -0300
Subject: Make gemspecs reproducible

With an explicit date, they will get the current date and make build
unreproducible
---
 ext/bigdecimal/bigdecimal.gemspec | 1 +
 ext/io/console/io-console.gemspec | 2 +-
 lib/rdoc/rdoc.gemspec             | 1 +
 3 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/ext/bigdecimal/bigdecimal.gemspec b/ext/bigdecimal/bigdecimal.gemspec
index 4efe438..c0d19e7 100644
--- a/ext/bigdecimal/bigdecimal.gemspec
+++ b/ext/bigdecimal/bigdecimal.gemspec
@@ -5,6 +5,7 @@
   s.name          = "bigdecimal"
   s.version       = _VERSION
   s.authors       = ["Kenta Murata", "Zachary Scott", "Shigeo Kobayashi"]
+  s.date          = RUBY_RELEASE_DATE
   s.email         = ["mrkn@mrkn.jp"]
 
   s.summary       = "Arbitrary-precision decimal floating-point number library."
diff --git a/ext/io/console/io-console.gemspec b/ext/io/console/io-console.gemspec
index 1256162..99e18bd 100644
--- a/ext/io/console/io-console.gemspec
+++ b/ext/io/console/io-console.gemspec
@@ -5,7 +5,7 @@
 Gem::Specification.new do |s|
   s.name = "io-console"
   s.version = _VERSION
-  s.date = date
+  s.date = RUBY_RELEASE_DATE
   s.summary = "Console interface"
   s.email = "nobu@ruby-lang.org"
   s.description = "add console capabilities to IO instances."
diff --git a/lib/rdoc/rdoc.gemspec b/lib/rdoc/rdoc.gemspec
index 2787f07..578c023 100644
--- a/lib/rdoc/rdoc.gemspec
+++ b/lib/rdoc/rdoc.gemspec
@@ -7,6 +7,7 @@
 
 Gem::Specification.new do |s|
   s.name = "rdoc"
+  s.date = RUBY_RELEASE_DATE
   s.version = RDoc::VERSION
   s.date = "2017-10-10"
 
